pom.xml

```diff
@@ -1235,6 +1235,10 @@
             <groupId>javax.annotation</groupId>
             <artifactId>javax.annotation-api</artifactId>
           </exclusion>
+          <exclusion>
+            <groupId>com.github.luben</groupId>
+            <artifactId>zstd-jni</artifactId>
+          </exclusion>
```

Comment:
Well, this is in my PR. Shall we merge mine and replace this?
- https://github.com/apache/spark/pull/31670/files

Reply:
Yes. This is just to verify whether it can succeed.